cartridges/int_algolia/cartridge/scripts/algolia/model/algoliaLocalizedProduct.js

```diff
@@ -466,6 +488,25 @@ var aggregatedValueHandlers = {
     },
     _tags: function(product) {
         return ['id:' + product.ID];
+    },
+    storeAvailability: function(product) {
+        var storeArray = [];
+        if (stores.length > 0) {
+            for (var i = 0; i < stores.length; i++) {
+                var storeEl = stores[i];
+                var storeElInventory = storeEl.storeInventory;
+                if (storeElInventory) {
+                    var inventoryRecord = storeElInventory.getRecord(product.ID);
+                    if (inventoryRecord && inventoryRecord.ATS.value && inventoryRecord.ATS.value > ALGOLIA_IN_STOCK_THRESHOLD) {
+                        storeArray.push(storeEl.id);
+                    }
+                }
+            }
+            if (storeArray.length > 0) {
+                return storeArray;
+            }
+        }
+        return undefined;
```

Comment:
Let's index an empty array by default so it's clearer that the field was indexed but no store had availability.
```suggestion
        return storeArray;
```